lib/utils/tls.go

```diff
@@ -35,15 +34,13 @@ func TLSConfig(cipherSuites []uint16) *tls.Config {
 
 // SetupTLSConfig sets up cipher suites in existing TLS config
 func SetupTLSConfig(config *tls.Config, cipherSuites []uint16) {
-	// If ciphers suites were passed in, use them. Otherwise use the the
+	// If cipher suites were passed in, use them. Otherwise, use the
 	// Go defaults.
 	if len(cipherSuites) > 0 {
 		config.CipherSuites = cipherSuites
 	}
 
 	config.MinVersion = tls.VersionTLS12
-	config.SessionTicketsDisabled = false
-	config.ClientSessionCache = tls.NewLRUClientSessionCache(DefaultLRUCapacity)
```

Comment:
Note the default for `SessionTicketsDisabled` is already `false`. This change will disable client session cache, but still support session tickets for anything using this config as a server. If the intent here is to disable session tickets altogether then we should put `config.SessionTicketsDisabled = true`

Reply:
If we decide to keep session tickets enabled for servers, then I think we should also keep the client session caching and just fix the tests to not clone the cache. That can be done in the test code by calling `SetupTLSConfig` after we clone the client, or by setting it to nil or something